cpp/src/arrow/array.h

```diff
@@ -275,11 +275,15 @@ class ARROW_EXPORT Array {
   /// This buffer does not account for any slice offset
   const uint8_t* null_bitmap_data() const { return null_bitmap_data_; }
 
+  /// Equality comparison with another array
   bool Equals(const Array& arr) const;
   bool Equals(const std::shared_ptr<Array>& arr) const;
 
-  bool ApproxEquals(const std::shared_ptr<Array>& arr) const;
-  bool ApproxEquals(const Array& arr) const;
+  /// Approximate equality comparison with another array
+  ///
+  /// epsilon is only used if this is FloatArray or DoubleArray
+  bool ApproxEquals(const std::shared_ptr<Array>& arr, double epsilon = 1E-5) const;
```

Comment:
Make the default a static constexpr `kDefaultEpsilon`.

Reply:
will do